playbooks/webapp/README.md

```diff
@@ -137,14 +137,14 @@ To delete the webapp:
 * **rds_instance_name** (str): Name for the database. Default: `mysampledb123`
 * **rds_engine** (str): Engine to use for the database. Default: `postgres`
 * **rds_engine_version** (str): Version number of the database engine to use. Default: `"14.8"`
-* **rds_master_user** (str): Name of the master user for the database instance. Default: `ansible`
+* **rds_master_username** (str): Name of the master user for the database instance. Default: `ansible`
```

Comment:
I guess this is a breaking change, do I?

Reply:
@alinabuzachis Yes it will be. It does seem like 'rds_master_username' would be a more appropriate variable name. What do you think?